arm-recoveryservicessiterecovery/2016-08-10/examples/ReplicationProtectedItems_TestFailover.json

```diff
@@ -12,7 +12,7 @@
         "failoverDirection": "PrimaryToRecovery",
         "networkType": "VmNetworkAsInput",
         "networkId": "/subscriptions/c183865e-6077-46f2-a3b1-deb0f4f4650a/resourceGroups/siterecoveryProd1/providers/Microsoft.Network/virtualNetworks/vnetavrai",
-        "skipTestFailoverCleanup": false,
+        "skipTestFailoverCleanup": "false",
```

Comment:
@veronicagg Service is expecting this as string. Its an intentional change. We realized this while trying to validate a variance of testfailover.